services/simcore/docker-compose.yml

```diff
@@ -321,6 +347,9 @@ services:
         delay: 5s
         max_attempts: 3
         window: 120s
+      placement:
+        constraints:
+        - node.role == manager
```

Comment:
as this is already in osparc-simcore you might not want to have it here. but it's not harming neither.

Reply:
removed